### DISCUSSION TITLE
Priorities 2024, 2025, and beyond

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#3] [High-level]
_We are currently collecting input via other [discussions](https://github.com/w3c/wai-mission/discussions/4) that will lead to proposals for:_

- What are top priorities
  - for 2024?
  - for 2025?
  - into 2030?
- What can be de-prioritized so effort can be spent on higher priorities?
- What are **risks** if we don't prioritize certain work?

**Comment by GeorgeKerscher** [comment]
Yes, prorities are essential. Where will the proposed priorities be posted so people can comment? Is it here?

**Comment by shawna-slh** [comment]
Yes, draft priorities will be provided -- after we process [other discussion points](https://github.com/w3c/wai-mission/discussions). Thanks.
